### PR TITLE
HBASE-23088 [hbase-thirdparty] Upgrade the dependencies for hbase-thi…

### PR DESCRIPTION
**Pull request by Apache9**
…rdparty

### CODE REVIEW

**Comment by Apache9** [comment]
> The commit message should include a summary of what dependecies changed to what versions.

Done.

File: hbase-shaded-netty/pom.xml [hunk 1]
```diff
@@ -141,7 +141,7 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-all</artifactId>
-      <version>4.1.34.Final</version>
+      <version>4.1.41.Final</version>
```
Comment (busbey) [inline hbase-shaded-netty/pom.xml]: 4.1.42 came out 3 days ago. Should use it?

Comment (saintstack) [reply hbase-shaded-netty/pom.xml]: Let me address as an addendum. Tried pushing here but no perms.

**Comment by saintstack** [comment]
Merged because I want a commit in hbase-thirdparty-3.1.0 else build tooling fails. Will address Sean comment in an addendum.

**Pull request merged**